src/main/java/org/folio/linked/data/service/impl/ResourceServiceImpl.java

```diff
@@ -75,13 +77,34 @@ public ResourceResponseDto createResource(ResourceRequestDto resourceDto) {
   }
 
   @Override
-  public Long createResource(org.folio.ld.dictionary.model.Resource modelResource) {
+  public Long saveMarcResource(org.folio.ld.dictionary.model.Resource modelResource) {
```

Comment:
Hi @PBobylev.
We need to add more log messages in this method and clearly log the inventory iD, resource ID etc. Else it will be difficult to trace in production.